main.tf

```diff
@@ -8,6 +8,7 @@ resource "null_resource" "docker_pullpush" {
     shell_hash = sha256("${var.docker_source}${var.ecr_repo_name}${var.ecr_repo_tag}")
   }
   provisioner "local-exec" {
+    interpreter = ["bash", "-c"]

```

Comment:
I wonder if `/bin/bash` is more canonical here?

Reply:
Hi!!

`/bin/bash` is specific of linux, it's a path to the executable file. I saw this and that's why I used `bash`;
https://github.com/hashicorp/terraform/issues/26305

Regards,
Javier.

Reply:
Thanks @jvillamonte - merged in now. Thanks for your contribution!

Reply:
Thank you for this module 🙃